<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MealPlanner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>perfil.ico</normaloff>perfil.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
color: white;
	font: 12pt &quot;Roboto&quot;;
	background-color: rgb(85, 170, 255);
}

QPushButton#btn_cancel {
	background-color: rgb(255, 70, 82);
}

QFrame{
	background-color: rgb(170, 170, 255);
}

QLabel#label_title{
	font: 87 20pt &quot;Roboto Black&quot;;
	color: rgb(0, 0, 127);
}

QLabel{
color: white;
	font: 10pt &quot;Roboto&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_foods">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>520</width>
      <height>381</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_food_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>29</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre Alimento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_protein_grams">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>101</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gramos Prote√≠nas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_carbos_grams">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>101</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gramos Carbohidratos:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsb_carbos_grams">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_food">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Alimento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_modify_food">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar Alimento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_food">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Alimento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>691</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="line_food_name">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_id">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>32</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_id">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_fat_grams">
     <property name="geometry">
      <rect>
       <x>489</x>
       <y>103</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gramos Grasa:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_fat_grams">
     <property name="geometry">
      <rect>
       <x>583</x>
       <y>102</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_protein_grams">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>103</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>90</y>
       <width>20</width>
       <height>601</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_food">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>610</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar Alimento</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_search_food">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>612</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_title">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MealPlanner</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_age">
     <property name="geometry">
      <rect>
       <x>734</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Edad:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_age">
     <property name="geometry">
      <rect>
       <x>774</x>
       <y>65</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_height">
     <property name="geometry">
      <rect>
       <x>857</x>
       <y>65</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Altura(cm):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_height">
     <property name="geometry">
      <rect>
       <x>936</x>
       <y>65</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_waist">
     <property name="geometry">
      <rect>
       <x>1005</x>
       <y>65</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cintura(cm):</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_waist">
     <property name="geometry">
      <rect>
       <x>1087</x>
       <y>65</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>220</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_activity_level">
     <property name="geometry">
      <rect>
       <x>826</x>
       <y>158</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nivel de Actividad:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_weight">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>111</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Peso(kg):</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_weight">
     <property name="geometry">
      <rect>
       <x>917</x>
       <y>110</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>30.000000000000000</double>
     </property>
     <property name="maximum">
      <double>300.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_neck">
     <property name="geometry">
      <rect>
       <x>775</x>
       <y>110</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_neck">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuello(cm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_goal">
     <property name="geometry">
      <rect>
       <x>1017</x>
       <y>113</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etapa:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_goal">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>110</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Volumen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mantenimiento</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Definicion</string>
      </property>
     </item>
    </widget>
    <widget class="QDoubleSpinBox" name="dsb_activity_level">
     <property name="geometry">
      <rect>
       <x>937</x>
       <y>157</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>1.200000000000000</double>
     </property>
     <property name="maximum">
      <double>2.200000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calculate_macros">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>200</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular Macros</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_macros_results">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>240</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_plate1">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>295</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Plato 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_f1_p1">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>322</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_f2_p1">
     <property name="geometry">
      <rect>
       <x>866</x>
       <y>321</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_f3_p1">
     <property name="geometry">
      <rect>
       <x>1025</x>
       <y>322</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 3:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a1_p1">
     <property name="geometry">
      <rect>
       <x>783</x>
       <y>319</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a2_p1">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>320</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a3_p1">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>321</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_a3_p2">
     <property name="geometry">
      <rect>
       <x>1025</x>
       <y>386</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_a1_p2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>386</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 1:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a1_p2">
     <property name="geometry">
      <rect>
       <x>783</x>
       <y>383</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a2_p2">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>384</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_plate2">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>360</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Plato 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_a2_p2">
     <property name="geometry">
      <rect>
       <x>866</x>
       <y>385</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 2:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a3_p2">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>385</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_a3_p3">
     <property name="geometry">
      <rect>
       <x>1025</x>
       <y>456</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_a1_p3">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>456</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 1:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a1_p3">
     <property name="geometry">
      <rect>
       <x>783</x>
       <y>453</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a2_p3">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>454</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_plate3">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>429</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Plato 3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_a2_p3">
     <property name="geometry">
      <rect>
       <x>866</x>
       <y>455</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 2:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a3_p3">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>455</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_a3_p4">
     <property name="geometry">
      <rect>
       <x>1025</x>
       <y>520</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_a1_p4">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>520</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 1:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a1_p4">
     <property name="geometry">
      <rect>
       <x>783</x>
       <y>517</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a2_p4">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>518</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_plate4">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>495</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Plato 4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_a2_p4">
     <property name="geometry">
      <rect>
       <x>866</x>
       <y>519</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Alimento 2:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_a3_p4">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>519</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_plates_results">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>600</y>
       <width>491</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_calculate_plates">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>560</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular Platos del D√≠a</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clean_search">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>610</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar B√∫squeda</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>table_foods</zorder>
   <zorder>label_food_name</zorder>
   <zorder>label_protein_grams</zorder>
   <zorder>label_carbos_grams</zorder>
   <zorder>btn_add_food</zorder>
   <zorder>btn_modify_food</zorder>
   <zorder>btn_delete_food</zorder>
   <zorder>btn_cancel</zorder>
   <zorder>dsb_carbos_grams</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
